apps/zui/src/models/lake.ts

```diff
@@ -1,7 +1,20 @@
 import {DomainModel} from "src/core/domain-model"
 import {LakeAttrs} from "../js/state/Lakes/types"
+import {Client} from "@brimdata/zed-js"
+import Slice from "src/js/state/Lakes"
+import LakeStatuses from "src/js/state/LakeStatuses"
+import {LakeStatus} from "src/js/state/LakeStatuses/types"
+import {FeatureDetector} from "./lake/feature-detector"
+
+type LakeModelAttrs = LakeAttrs & {status: LakeStatus}
+
+export class Lake extends DomainModel<LakeModelAttrs> {
```

Comment:
Here we are adding the methods we need to the Lake model. 

I've added:

- find(id)
- client 
- features
- sync()
- update
- save
- isRunning()